deploy/helm/secret-operator/values.yaml

```diff
@@ -6,8 +6,10 @@ image:
   pullSecrets: []
 
 secretMigrationJob:
+  enabled: true
   image:
     repository: docker.stackable.tech/stackable/tools
+    tag: 1.0.0-stackable24.11.0
```

Comment:
I hard-coded the default value to `1.0.0-stackable24.11.0` because if the tag depends on the `appVersion`, a dependency between the secret-operator image and the tools image would be created.

For instance, if the secret-operator is installed in version `0.0.0-pr536` then a tools image with the tag `1.0.0-stackable0.0.0-pr536` would also be required.